README.md

```diff
@@ -1413,10 +1413,10 @@ If you add a dependency on a package with native code, you need to build the
 wheel manually:
 
 ```
-$ docker run -it -v ${azul_home}/:/root/azul python:3.6.8-stretch bash
+$ docker run -it -v ${azul_home}/:/root/azul python:3.8.2-stretch bash
```

Comment:
Be sure to try if this still works.

Reply:
My mistake, I didn't read the documentation above. I have since re-build the wheel and replaced the old one in source control. I also updated this log.